app/src/androidTest/java/com/github/orkest/View/sharedMusic/SharedMusicPostUITest.kt

```diff
@@ -0,0 +1,90 @@
+package com.github.orkest.View.sharedMusic
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.orkest.Model.Profile
+import com.github.orkest.Model.Song
+import com.github.orkest.View.sharing.ui.theme.OrkestTheme
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class SharedMusicPostUITest {
+
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    @Before
+    fun setUp(){
+        val song = Song("Bad boy", "Rihanna", "Rated R")

```

Comment:
This song was added as a constant in the Constants file in order to use it directly in tests

Reply:
Thank you for letting me know. I updated the code